packages/web/src/components/withdraw-usdc-modal/WithdrawUSDCModal.tsx

```diff
@@ -65,6 +76,16 @@ export const WithdrawUSDCModal = () => {
   const balanceNumberCents = formatUSDCWeiToFloorCentsNumber(
     (balance ?? new BN(0)) as BNUSDC
   )
+  const withdrawalStatus = useSelector(getWithdrawStatus)
+
+  const [priorBalanceCents, setPriorBalanceCents] =
+    useState<Nullable<number>>(null)
+
+  useEffect(() => {
```

Comment:
do we need an effect here? can we just have the default for `priorBalanceCents` be `balanceNumberCents` instead of null? (And maybe even use a ref?)

Reply:
balanceNumberCents won't necessarily be defined correctly initially (or it isn't a requirement to), so there does need to be some kinda setter thing for it.